components/renku_data_services/namespace/blueprints.py

```diff
@@ -163,7 +163,7 @@ async def _get_namespaces(
                     name=ns.name,
                     slug=ns.latest_slug if ns.latest_slug else ns.slug,
                     created_by=ns.created_by,
-                    creation_date=ns.creation_date,
+                    creation_date=None,  # NOTE: we do not save creation date in the DB
```

Comment:
Maybe we should keep an issue to fix the API spec then?

Reply:
https://github.com/SwissDataScienceCenter/renku-data-services/issues/283